src/redux/books/booksSlice.js

```diff
@@ -1,41 +1,108 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
 
-const booksSlice = createSlice({
-  name: 'books',
-  initialState: [
-    {
-      id: 'item1',
-      title: 'The Great Gatsby',
-      author: 'John Smith',
-      category: 'Fiction',
-    },
-    {
-      id: 'item2',
-      title: 'Anna Karenina',
-      author: 'Leo Tolstoy',
-      category: 'Fiction',
+const appId = 'K1k1N3RPPBNvJlgdPD3o';
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps';
+
+const initialState = {
+  books: [],
+  isLoading: false,
+  error: null,
+};
+
+export const fetchBooks = createAsyncThunk('books/fetchBooks', async (thunkAPI) => {
+  try {
+    const response = await axios.get(`${baseUrl}/${appId}/books`);
+    // console.log(response);
```

Comment:
- [Optional] Please comments render codebase messy. Let's not push commented codes to the repo. Please remove before merging